samples/bluetooth/broadcast_audio_source/boards/nrf52840dk_nrf52840.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_BT_HCI_TX_STACK_SIZE_WITH_PROMPT=y
+CONFIG_BT_HCI_TX_STACK_SIZE=940
```

Comment:
The stack size profiling for nRF53 is here: https://github.com/zephyrproject-rtos/zephyr/pull/56687

We can add something like below in Kconfig file
```
	default 1152 if BT_CTLR && BT_LL_SW_SPLIT && BT_ISO_BROADCAST
```

Reply:
Actually, the minimum stack size we need is **886**. If we can agree on a number, I would be happy to implement it per default like @cvinayak suggested. This would be way nicer than the board file.

Reply:
@GatCode Could you update this PR as per suggested by @cvinayak ? 

I suggest to add a stack size that may be slightly larger than the minimum you quoted (886). I suggest 940

Reply:
@Thalley, happy to but can we please agree on a stack size value? You argued ** 832**, @cvinayak said **1152** and the actual minimum is **886**.

Reply:
I would suggest 940, which is slightly larger than the minimum